### PR TITLE
Make Kotlin upgrade easier

### PR DESCRIPTION
**Pull request by rachelcarmena**
- Gradle upgrade to be able to use a variable for dependency version.
- Replace hard-coded version by the correspondent variable.
- Add test property to get Kotlin version from test code.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -75,3 +75,7 @@ compileTestKotlin {
     kotlinOptions.jvmTarget = "1.8"
     kotlinOptions.freeCompilerArgs += ["-Xskip-runtime-version-check"]
 }
+
+test {
+    systemProperty "KOTLIN_VERSION", "$kotlin_version"
```
Comment (tschuchortdev) [inline build.gradle:80]: Am I correct to assume that this sets an environment variable `KOTLIN_VERSION` to the current value of the Gradle variable `kotlin_version`?

Comment (rachelcarmena) [reply build.gradle:80]: Right, to be used here: https://github.com/tschuchortdev/kotlin-compile-testing/pull/42/files/3c55b819e39078b5e7f5c7c752a1c787a71bf257#diff-6efbdba372511a9217d44dd2924076e0

Link: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.testing.Test.html

Comment (tschuchortdev) [reply build.gradle:80]: Alright, I'll merge it then.

**Comment by tschuchortdev** [comment]
Thanks for your contribution!

**Comment by rachelcarmena** [comment]
> Thanks for your contribution!

Thanks to you for your useful library :raised_hands:

**Pull request merged**